### PR TITLE
showTrue creation count , make the other user profile color change , …

### PR DESCRIPTION
**Pull request by UmairMukhtar**
…unverified profile show show different icon the button for other profiel should not display
image

unverifed symbol

### CODE REVIEW

**Comment by UmairMukhtar** [comment]
https://github.com/eLearningDAO/POCRE/issues/285

File: app/web-frontend/src/pages/Wallet/common/components/useProfile.jsx [hunk 1]
```diff
@@ -52,8 +55,36 @@ const useProfile = () => {
     },
   });
 
+  const {
+    data: creations,
+    isError: isFetchError,
+    isSuccess: isFetchSuccess,
+  } = useQuery({
+    queryKey: ['creations'],
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/web-frontend/src/pages/Wallet/common/components/useProfile.jsx]: done

File: app/web-frontend/src/pages/Wallet/common/components/profile.jsx [hunk 2]
```diff
@@ -89,7 +97,7 @@ function WalletProfile({
           <p className="wallet-profile-creations-count">
             <Button onClick={handleViewCreattion} style={{ color: 'black' }}>
               Author of
-              <span style={{ marginLeft: '4px', marginRight: '4px' }}>{totalCreationsAuthored}</span>
+              <span style={{ marginLeft: '4px', marginRight: '4px' }}>{totalCreationsAuthored || creationCount}</span>
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/web-frontend/src/pages/Wallet/common/components/profile.jsx]: done

**Comment by AltiMario** [comment]
On click to the confirmation
Screenshot_20230126_133244
I received this error 
Screenshot_20230126_133209
and the profile was still "in pending"

**Review by AltiMario — APPROVED** [review]

**Pull request merged**